relay-profiling/src/utils.rs

```diff
@@ -47,3 +49,23 @@ where
 pub fn string_is_null_or_empty(s: &Option<String>) -> bool {
     s.as_deref().map_or(true, |s| s.is_empty())
 }
+
+pub fn default_client_sdk(platform: &str) -> ClientSdk {
+    let sdk_name = match platform {
+        "android" => "sentry.java.android",
+        "cocoa" => "sentry.cocoa",
+        "csharp" => "sentry.dotnet",
+        "go" => "sentry.go",
+        "javascript" => "sentry.javascript",
+        "node" => "sentry.javascript.node",
+        "php" => "sentry.php",
+        "python" => "sentry.python",
+        "ruby" => "sentry.ruby",
+        "rust" => "sentry.rust",
+        _ => "unknown",
+    };
+    ClientSdk {
+        name: sdk_name.into(),
+        version: "unknown".into(),
+    }
+}
```

Comment:
Wouldnt it be better to return `Option<ClientSdk>` and use `None` for the `unknown` cases instead?